index.html

```diff
@@ -10,147 +10,182 @@
-          <a href="#" rel="noopener noreferrer" class="works-link">
-            <div class="works-button">
-              <span>See this project</span>
-              <div class="works-arrow"></div>
+          </div>
+          <div class="project-5">
+            <img class="works-img" src="./Media/work-img-5.svg" alt="Man typing on a laptop">
+            <div class="works-box">
+              <h3>Project name goes here</h3>
+              <ul class="lang-container">
+                <li class="lang-box">HTML/CSS</li>
+                <li class="lang-box">Ruby on Rails</li>
+                <li class="lang-box">JavaScript</li>
+              </ul>
+              <a href="#" rel="noopener noreferrer" class="works-link">
+                <div class="works-button">
+                  <span>See this project</span>
+                  <div class="works-arrow"></div>
+                </div>
+              </a>
             </div>
-          </a>
+          </div>
         </div>
       </li>
-      <li>
-        <img class="works-img" src="./Media/work-img-6.svg" alt="Man typing on a laptop">
-        <div class="works-box">
-          <h3>Project name goes here</h3>
-          <ul class="lang-container">
-            <li class="lang-box">HTML/CSS</li>
-            <li class="lang-box">Ruby on Rails</li>
-            <li class="lang-box">JavaScript</li>
-          </ul>
-          <a href="#" rel="noopener noreferrer" class="works-link">
-            <div class="works-button">
-              <span>See this project</span>
-              <div class="works-arrow"></div>
-            </div>
-          </a>
+      <li class="c4">
+        <div class="deco-container">
+          <img class="deco-4" src="./Media/image-geometry_5.svg" alt=" ">
+          <img class="deco-5" src="./Media/image-geometry_4.svg" alt=" ">
+        </div>
+        <div class="project-6">
+          <img class="works-img" src="./Media/work-img-6.svg" alt="Man typing on a laptop">
+          <div class="works-box">
+            <h3>Project name goes here</h3>
+            <ul class="lang-container">
+              <li class="lang-box">HTML/CSS</li>
+              <li class="lang-box">Ruby on Rails</li>
+              <li class="lang-box">JavaScript</li>
+            </ul>
+            <a href="#" rel="noopener noreferrer" class="works-link">
+              <div class="works-button">
+                <span>See this project</span>
+                <div class="works-arrow"></div>
+              </div>
+            </a>
+          </div>
         </div>
       </li>
     </ul>
   </section>
-  <section class="about">
-    <h2>About</h2>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
-      eiusmod
-    </p>
-    <div class="about-container">
+  <section id="about-wrapper" class="about">
```

Comment:
The space between the image and the text should be reduced 
### Original design
![right](https://user-images.githubusercontent.com/58376020/125847547-68745f29-69ac-4956-a4e1-64b0ba979a24.png)

#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125847589-aa7baf03-d0d9-48e1-9d19-eb2c6abc1904.png)

Reply:
Change applied successfully.